app/controllers/concerto_weather/search_controller.rb

```diff
@@ -12,9 +12,11 @@ def find_city
       def getOpenWeatherCities(query)
         require 'net/http'
         require 'json'
+        require 'erb'
```

Comment:
@garci66 would you mind using [URI.escape](https://ruby-doc.org/stdlib-2.2.0/libdoc/uri/rdoc/URI/Escape.html#method-i-escape) instead?